src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt

```diff
@@ -62,6 +62,7 @@ class JoarkMottak(val env: Environment, private val journalpostArkiv: Journalpos
 
         inngåendeJournalposter
                 .peek { key, value -> LOGGER.info("Processing ${value.javaClass} with key $key") }
+                .filter { _, journalpostHendelse -> "DAG" == journalpostHendelse.get("temaNytt").toString() }
```

Comment:
I'll need to figure out if "temaNytt" or "temaGammelt" is the field to filter out

Reply:
`temaNytt` should be correct. This is for the cases where the `tema` is changed, usually by a case worker, because it wrong the first time around. We still need to pick those up, so we should only process those that change (or are created) with `tema = DAG`.

Reply:
Have asked "team dokument"